U/Uno/build_tarballs.jl

```diff
@@ -0,0 +1,64 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Uno"
+
+version = v"1.1.0"
+
+sources = [
+    GitSource(
+        "https://github.com/cvanaret/Uno.git",
+        "d91ed34d5e25c3c16ebc34a7c5a67b9c35c2d33f",
+    ),
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir/Uno
+mkdir -p build
+cd build
+
+cmake \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBLA_VENDOR="libblastrampoline" \
+    -DMUMPS_INCLUDE_DIR=${includedir} \
+    -DMUMPS_LIBRARY=${libdir} \
+    ..
+
+make -j${nproc}
+make install
+"""
+
+platforms = supported_platforms()
+
+products = [
+    ExecutableProduct("Uno", :amplexe),
+]
+
+dependencies = [
+    Dependency(PackageSpec(name="ASL_jll", uuid="ae81ac8f-d209-56e5-92de-9978fef736f9"), compat="0.1.3"),
+    Dependency(PackageSpec(name="MUMPS_seq_jll", uuid="d7ed1dd3-d0ae-5e8e-bfb4-87a502085b8d")),
+    Dependency(PackageSpec(name="libblastrampoline_jll", uuid="8e850b90-86db-534c-a0d3-1478176c7d93"), compat="5.4.0"),
+    # For OpenMP we use libomp from `LLVMOpenMP_jll` where we use LLVM as compiler (BSD
+    # systems), and libgomp from `CompilerSupportLibraries_jll` everywhere else.
+    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"); platforms=filter(!Sys.isbsd, platforms)),
+    Dependency(PackageSpec(name="LLVMOpenMP_jll", uuid="1d63c593-3942-5779-bab2-d838dc0a180e"); platforms=filter(Sys.isbsd, platforms)),
```

Comment:
Does this work as intended? The final `Project.toml` can only have a single fixed list of dependencies, not different ones for BSD vs. non-BSD systems.

Reply:
IIUC: both get added to Project.toml, but the JLL init script will load different things on different platforms?

Reply:
This is the standard way of representing this. Inside the init scripts of a package, it then only loads the required ones (e.g., see the SZ_jll scripts for macOS here: https://github.com/JuliaBinaryWrappers/SZ_jll.jl/blob/main/src/wrappers/aarch64-apple-darwin.jl and for Linux here: https://github.com/JuliaBinaryWrappers/SZ_jll.jl/blob/main/src/wrappers/x86_64-linux-gnu.jl).